woocommerce/woocommerce-ios
# Remove `multipleSelectionsEnabled` property from ProductSelectorView's Configuration

Closes: #9435 

## Description
The `ProductSelectorView` configuration has a `multipleSelectionsEnabled` property which is no longer necessary. This PR removes it.

## Context
The property was initially used to differentiate between product single-selection (Order Creation), and product multi-selection (Order Creation, and Coupons). Since we've moved to use only product multi-selection on Order creation, multiple selection is always enabled and the property is no longer needed.

We still keep the property for `ProductRow` as this has different use cases and needs to render the rows differently when needed.

## Testing instructions
- Order Creation: Go to Orders > `+` > `+ Add Products` > Confirm that you can multi-select and the confirmation button ("_X products selected_") renders at the bottom of the product selector.
- Coupons: Go to Menu > Settings > Experimental Features > Enable Coupon Management > Menu > Coupons > `+` > Tap on `Percentage discount` > Tap `All Products` > Confirm that you can multi-select

## Screenshots
| Product selector | Variation selector|
|--------|--------|
| ![Simulator Screen Shot - iPhone 11 - 2023-04-25 at 14 03 30](https://user-images.githubusercontent.com/3812076/234200080-e964f201-162e-4f56-8cfa-9c8a8593b560.png) | ![Simulator Screen Shot - iPhone 11 - 2023-04-25 at 14 04 53](https://user-images.githubusercontent.com/3812076/234200008-b5192ba7-138b-43aa-8ff4-9d6e29dc1bd4.png) |
